tests/tests_hrv.py

```diff
@@ -102,14 +102,21 @@ def test_hrv_interpolated_rri(interpolation_rate):
     peaks = peaks["ECG_R_Peaks"]
     rri = np.diff(peaks).astype(float)
     rri_time = peaks[1:] / 1000
-    
+
     if interpolation_rate=="from_mean_rri":
         interpolation_rate = 1000/np.mean(rri)
+
     rri_processed, _ = nk.intervals_process(
         rri, intervals_time=rri_time, interpolate=True, interpolation_rate=interpolation_rate
     )
 
-    ecg_hrv = nk.hrv({"RRI": rri_processed, "RRI_Time": rri_time})
+    rri_processed_time = np.arange(
```

Comment:
I wonder if `intervals_process()` should return the new timestamps so that they don't have to be recomputed outside of the function. Maybe only if `intervals_time` is passed?

Reply:
I'd say you know best :)